### ISSUE TITLE
[14r] Can't walk up stairs or slabs properly.

### ISSUE THREAD

**Issue by bigolfatty** [APengu/LostThaumaturgy#37]
When walking into stairs or slabs next to a dirt path, instead of ascending like normal, I get caught on them and have to jump.

**Comment by APengu** [comment]
Is it plain Lost Thaum and Hammer Core? Other mods may be messing with that

**Comment by bigolfatty** [comment]
Yeah, I disabled all other mods but those and I'm running the 1.12.1 recommended version of forge. It didn't happen with only hammer core, but when i added lost thaum it did. Maybe it has something to do with the boots of the traveler?

**Comment by APengu** [comment]
Did this happen before?

**Comment by bigolfatty** [comment]
I just found out about this mod yesterday, so this is the first version I've tried. I've been playing with a number of other mods the past few weeks though, and this issue never appeared then.
